packages/evolution-legacy/src/styles/survey/_header.scss

```diff
@@ -10,18 +10,27 @@ header {
     display: flex;
     justify-content: space-between;
     padding: $xxs-size;
+
+    // Ensures the title text in the top navigation bar doesn't overflow
     h1 {
       opacity: 0.6;
       margin: 0;
+      width: 100%;
+      max-width: 180px;
```

Comment:
On normal browsers is there a chance that the text will be shown with ellipsis even if there is enough space?

Reply:
No, it will only show an ellipsis if there is not enough space.

Reply:
Yes, with this setup, after 180px. 

![image](https://github.com/chairemobilite/evolution/assets/78976679/71dcc2ec-bbad-4d21-adea-cb76a908b688)